### PR TITLE
Add option to hide the upload estimation in WebUI

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
The upload estimation can now be hidden when setting `hide_upload_estimation` to "yes" via occ command.

## Related Issue
- Fixes https://github.com/owncloud/enterprise/issues/4743

## Screenshots (if appropriate):

image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 
- [x] Changelog item, see [TEMPLATE](https://github.com/owncloud/core/blob/master/changelog/TEMPLATE)

### CODE REVIEW

File: changelog/unreleased/39228 [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Add option to hide the upload estimation in WebUI
+
+The upload estimation can now be hidden when setting
+`hide_upload_estimation` to "yes" via occ command.
```
Comment (phil-davis) [inline changelog/unreleased/39228:4]: To test, I did:
`occ config:app:set files hide_upload_estimation --value="yes"`

And then it works - there is no time estimation, just "Uploading..." and then "Processing files..."

Should we write the exact command in the changelog entry?

**Review by phil-davis — APPROVED** [review]
works

**Comment by phil-davis** [comment]
Note: there are not automated acceptance tests for this sort of thing, because the progress bar is very timing-dependent, and tests might or might not "see" what is happening in the progress bar, depending on if the test runner and/or system-under-test are running on fast or slow systems.

**Review by AlexAndBear — APPROVED** [review]

**Comment by JanAckermann** [comment]
Docs relevant, 

please file a docs issue.

**Comment by phil-davis** [comment]
> Docs relevant,
> 
> please file a docs issue.

https://github.com/owncloud/docs/issues/4034

**Comment by GeraldLeikam** [comment]
works. For the future: Please write the complete occ command in the description. It would save us a lot of testing time if we had the command directly and didn't have to search for it and assemble it. thanks

**Pull request merged**